joinmarket/blockchaininterface.py

```diff
@@ -749,13 +758,13 @@ def __init__(self, jsonRpc):
         super(RegtestBitcoinCoreInterface, self).__init__(jsonRpc, 'regtest')
         self.pushtx_failure_prob = 0
         self.tick_forward_chain_interval = 2
-	self.absurd_fees = False
+        self.absurd_fees = False
 
-    def estimate_fee_per_kb(self, N):
-	if not self.absurd_fees:
-	    return super(RegtestBitcoinCoreInterface, self).estimate_fee_per_kb(N)
-	else:
-	    return jm_single().config.getint("POLICY", "absurd_fee_per_kb") + 100
+     def estimate_fee_per_kb(self, N):
```

Comment:
Seems to be 4 spaces-> 5 ? I think this might be why my editor complains about this particular file's formatting.